apps/dg/components/case_card/column_resize_handle.js

```diff
@@ -0,0 +1,146 @@
+/* global React */
+
+DG.React.ready(function () {
+  var div = React.DOM.div;
+
+  /**
+   * Provides a vertical column-resizing control handle.
+   * Loosely patterned after https://github.com/MonsantoCo/column-resizer.
+   */
+  DG.React.Components.ColumnResizeHandle = DG.React.createComponent(
+    (function () {
+
+      /**
+       * props are
+       *  className: {string} - className attribute applied to the dom element
+       *  enabled: {boolean} - enable/disable resizing
+       *  containerWidth: {number} - width of table in pixels
+       *  columnWidth: {number} - width of column to resize in pixels
+       *  minWidth: {number} - minimum width of column in pixels
+       *  maxWidth: {number} - maximum width of column in pixels
+       *  onResize(width: number): {function} - Called with updated value of width property
+       */
+      var resizeTouchId = null,
+          deltaStart = null;
+
+      function stop(e) {
+        e.preventDefault();
+        e.stopPropagation();
+      }
+
+      return {
+
+        getInitialState: function() {
+          return {
+            isResizing: false,
+            resizeWidth: null
+          };
+        },
+
+        componentDidMount: function() {
+          this.componentDidRender();
+        },
+
+        componentDidUpdate: function() {
+          this.componentDidRender();
+        },
+
+        componentDidRender: function() {
+          if (this.state.resizeWidth !== this.props.columnWidth)
+            this.setState({ resizeWidth: this.props.columnWidth });
+        },
+
+        render: function() {
+          var style = this.state.resizeWidth != null
+                        ? { left: this.state.resizeWidth }
+                        : null;
+          return (this.props.enabled || this.state.isResizing) &&
+            div({
+              className: this.props.className + ' column-resize-handle dg-wants-mouse dg-wants-touch',
+              onMouseDown: this.handleMouseDown,
+              onTouchStart: this.handleInitialTouch,
```

Comment:
The initial `mousedown` and `touchstart` handlers are installed as React handlers, which means that React is responsible for cleaning them up when the component is unmounted. The subsequent handlers are installed directly on the window, so we're responsible for cleaning them up.